NodeDependencyLookup/Editor/Caches/AssetDependencyCache/AssetTraverser/AssetSerializedPropertyTraverser.cs

```diff
@@ -10,9 +10,9 @@ namespace Com.Innogames.Core.Frontend.NodeDependencyLookup
 {
 	public class AssetSerializedPropertyTraverser : AssetTraverser
 	{
-		private class ReflectionStackItem
+		private struct ReflectionStackItem
```

Comment:
Why did you decide to use a struct here? I think from the data types inside (e.g. `object`), a class is a better choice (see also [MSDN guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/choosing-between-class-and-struct?redirectedfrom=MSDN))